lib/membrane_flac_plugin/membrane_flac_parser.ex

```diff
@@ -38,18 +47,43 @@ defmodule Membrane.FLAC.Parser do
     {[], state}
   end
 
+  defp calculate_pts(buffer, state) do
+    if state.generate_best_effort_timestamps? do
+      pts =
+        case buffer.metadata do
+          %{sample_rate: sample_rate, starting_sample_number: starting_sample_number} ->
+            (starting_sample_number / sample_rate * 1_000_000_000) |> trunc()
+
+          _credo_silencer ->
+            nil
+        end
+
+      Map.merge(buffer, %{pts: pts})
```

Comment:
This is slightly more Elixir way
```suggestion
      %Buffer{buffer | pts: pts}
```